src/components/PromptBuilder/styles.module.css

```
@@ -0,0 +1,321 @@
+  color: var(--ifm-color-primary);
+}
+
+.previewBody {
+  margin: 0;
+  padding: 1.25rem;
+  font-family: var(--ifm-font-family-monospace);
+  font-size: 0.8rem;
+  color: #d0d0d0;
+  line-height: 1.65;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)